bin/node/runtime/src/lib.rs

```diff
@@ -131,7 +131,7 @@ impl frame_system::Trait for Runtime {
 	type ModuleToIndex = ModuleToIndex;
 	type AccountData = pallet_balances::AccountData<Balance>;
 	type OnNewAccount = ();
-	type OnReapAccount = (Balances, Staking, Contracts, Session, Recovery);
+	type OnReapAccount = ();
```

Comment:
from the other PR: 

> system::Trait is now the place to put any OnNewAccount or OnReapAccount hooks; these will need moving from balances::Trait. You should include _at least_ the Balances handler in the reap account

Reply:
Only if you store your balances in the balances module I think.

If it is stored in the composite account, then you wont need to.

let me double check